libs/components/ag-grid/src/lib/modules/ag-grid/types/cell-type.ts

```diff
@@ -40,6 +40,15 @@ export enum SkyCellType {
    * Specifies that cells in the column will display the currently selected date formatted as `MM-DD-YYYY`, or the date format of the locale passed to `getGridOptions()`. If you would like to overwrite this format, you can [define a `valueFormatter`](https://www.ag-grid.com/javascript-grid-value-formatters/) on the column definition. See the demo for an example.
    */
   Date = 'skyCellDate',
+  /**
+   * **Edit mode**
+   * <br/>
+   * Specifies that cells in the column will be edited as  [SKY UX lookup components](https://developer.blackbaud.com/skyux-v5/components/lookup). You can set any of the lookup component's properties by passing them in the [column definition's `cellEditorParams` property](https://www.ag-grid.com/javascript-grid-column-properties/). These params can be updated as other cell edits are made or [provided dynamically](https://www.ag-grid.com/javascript-grid-cell-editing/#dynamic-parameters) based on other cell values. See the demo for an example. Text can be entered and a value selected from the provided list.
```

Comment:
There looks to be an extra space before the `SKY UX lookup components` link

Reply:
* Link to the AG Grid Angular docs e.g. `https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing` rather than "javascript"
* Reference the `SkyCellEditorLookupParams` type as what to pass to `cellEditorParams` — when typedoc is wired up to [the development tab](https://developer.blackbaud.com/skyux-v5/components/data-entry-grid?docs-active-tab=development) we'll be able to link to that type

Reply:
@johnhwhite I can replace the https://www.ag-grid.com/javascript-grid-column-properties/ url with https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing in this JSDoc comment, but it also exists in all of the other cell type's JSDoc comments. Should I replace them there, as well?